hs_discover/src/components/Resources.vue

```diff
@@ -0,0 +1,573 @@
+<template>
```

Comment:
self-documenting, well-named variables, comments in code, html and even CSS! strong typing, tight linting, even the whitespace!

Reply:
conforms entirely to eslint, much tighter and useful than the sonarqube results for frontend code, which should be disregarded

Reply:
Pretty sure I voiced this last time I reviewed.  I would have expected the Discover page to be broken up into several components.  It looks like the discover page would break up into a map component, a facet component (and each facet would use the same component with different datasets), and a resource listing component.

I, however, have little experience with Vue outside of a couple tutorials.  I won't hold this PR up due to the Vue, but I maintain my concern.